Packages/com.github.homuler.mediapipe/Runtime/Scripts/Unity/Experimental/TextureFrame.cs

```diff
@@ -93,6 +93,7 @@ public void Dispose()
       }
       _glSyncToken?.Dispose();
       _ = _InstanceTable.Remove(_instanceId);
+      MonoBehaviour.Destroy(_texture);
```

Comment:
Since the `Destroy` method is defined in `UnityEngine.Object`, will you rewrite this line as follows?
```suggestion
      UnityEngine.Object.Destroy(_texture);
```